README.rst

```diff
@@ -46,7 +47,7 @@ Otherwise to get voice support you should run the following command:
     py -3 -m pip install -U py-cord[voice]
 
 
-To install the development version, do the following:
+To install the latest development version, do the following:
```

Comment:
development version is not latest. latest should be considered the stable version and development version is usually not at stable state and shouldn't be encouraged to be installed.